PurchasesTests/Misc/AttributionFetcherTests.swift

```diff
@@ -55,5 +65,64 @@ class AttributionFetcherTests: XCTestCase {
         
         expect { self.attributionFetcher.rot13(randomized) } .to(equal(expected))
     }
+    
+    func testPostAttributionDataSkipsIfAlreadySent() {
```

Comment:
I overestimated how tested this was - I thought that tests passing for the refactor meant that we were all good, but there weren't really any tests for this stuff.